### PR TITLE
feat: support user config of API endpoint

### LINKED ISSUE #15: Configure API endpoint URL
* define via env var
* maybe define in method params too

### PR DESCRIPTION
**Pull request by jsstevenson**
close #15

### CODE REVIEW

File: src/dgipy/dgidb.py [hunk 1]
```diff
@@ -1,25 +1,11 @@
...
-        url = "https://staging.dgidb.org/api/graphql"
-
-    if env == "production":
-        url = "https://dgidb.org/api/graphql"
-
-    return url
-
-
-base_url = __api_url("local")
+API_ENDPOINT_URL = os.environ.get("DGIDB_API_URL", "https://dgidb.org/api/graphql")
```
Comment (mcannon068nw) [inline src/dgipy/dgidb.py:9]: out of curiosity, does this check for a global variable by this name, then if none exists, default to the second arg?

**Review by mcannon068nw — APPROVED** [review]
seems straightforward and efficient 👍

File: src/dgipy/dgidb.py (hunk 1 above)
Comment (jsstevenson) [reply src/dgipy/dgidb.py:9]: @mcannon068nw exactly. `os.environ` is just a `Dict`, and the dict's `get()` method is like a safe way to index into a dictionary. By default it returns None if the key isn't in the dictionary but you can give a default value as the second parameter.

Comment (jsstevenson) [reply src/dgipy/dgidb.py:9]: And to be more specific: it's check for a UNIX environment variable, which is a common way of setting custom configurations https://www.geeksforgeeks.org/environment-variables-in-linux-unix/amp/

**Pull request merged**